docs/site/output-webview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: The Output Webview
+description: A read-only webview for REPL output.
+---
+
+The output webview is a read-only webview for REPL output. It's more performant than [the editor-based REPL window](repl-window.md).
```

Comment:
It has a ton of other benefits too. As well as things where it loses. (Such as paredit navigation and selection, which we probably should add later).

I think we should have a small list of features here (such as rich stack traces) and link to the output destinations page for a discussion about how the different destinations compare.